docker-compose.gha.yml

```diff
@@ -21,14 +21,22 @@ services:
       - ./dtfs-sql-volume-dtfs-submissions:/data/db
 
   dtfs-submissions-data:
-    image: mongo:5.0.5
-    restart: always
+    build:
+      context: ./mongodb
+      dockerfile: Dockerfile
     environment:
       - MONGO_INITDB_DATABASE=dtfs-submissions
       - MONGO_INITDB_ROOT_USERNAME=root
       - MONGO_INITDB_ROOT_PASSWORD=r00t
     ports:
-      - '27017-27019:27017-27019'
+      - "27017:27017"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    healthcheck:
```

Comment:
What does this `healthcheck` `test` do? And is it safe to be echoing username/password?